### PR TITLE
fix download dataset

### PR DESCRIPTION
**Pull request by caiyinqiong**

### CODE REVIEW

File: matchzoo/datasets/quora_qp/load_data.py [hunk 1]
```diff
@@ -55,37 +53,17 @@ def load_data(
...
+        'quora_qp', _url, extract=True,
+        cache_dir=matchzoo.USER_DATA_DIR,
+        cache_subdir='quora_qp'
+    )
+    return Path(ref_path).parent.joinpath('QQP')
 
 
 def _read_data(path, stage):
-    data = pd.read_csv(path, sep='\t', error_bad_lines=False, dtype=object)
+    data = pd.read_csv(path, sep='\t', error_bad_lines=False)
```
Comment (Chriskuei) [inline matchzoo/datasets/quora_qp/load_data.py]: why delete `dtype=object`?

File: matchzoo/utils/get_file.py [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+                                       current - self._seen_so_far]
+                else:
+                    self._values[k][0] += v * (current - self._seen_so_far)
+                    self._values[k][1] += (current - self._seen_so_far)
+            else:
+                self._values[k] = [v, 1]
+        self._seen_so_far = current
+
+        now = time.time()
+        info = ' - %.0fs' % (now - self._start)
```
Comment (Chriskuei) [inline matchzoo/utils/get_file.py]: Convert `%` to py3 style

File: matchzoo/utils/get_file.py [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+
+            self._total_width = len(bar)
+            sys.stdout.write(bar)
+
+            if current:
+                time_per_unit = (now - self._start) / current
+            else:
+                time_per_unit = 0
+            if self.target is not None and current < self.target:
+                eta = time_per_unit * (self.target - current)
```
Comment (Chriskuei) [inline matchzoo/utils/get_file.py]: Wrapped in `matchzoo.utils.Timer`

File: matchzoo/utils/get_file.py [hunk 4]
```diff
@@ -0,0 +1,425 @@
...
+                        if os.path.isfile(path):
+                            os.remove(path)
+                        else:
+                            shutil.rmtree(path)
+                    raise
+            return True
+    return False
+
+
+def get_file(fname,
```
Comment (Chriskuei) [inline matchzoo/utils/get_file.py]: Use `typing` to format code

File: matchzoo/utils/get_file.py [hunk 5]
```diff
@@ -0,0 +1,425 @@
...
+                print('A local file was found, but it seems to be '
+                      'incomplete or outdated because the file hash '
+                      'does not match the original value of file_hash.'
+                      ' We will re-download the data.')
+                download = True
+    else:
+        download = True
+
+    if download:
+        print('Downloading data from', origin)
```
Comment (Chriskuei) [inline matchzoo/utils/get_file.py]: Need a space?

File: matchzoo/datasets/quora_qp/load_data.py (hunk 1 above)
Comment (caiyinqiong) [reply matchzoo/datasets/quora_qp/load_data.py]: have added it.

File: matchzoo/utils/get_file.py (hunk 2 above)
Comment (caiyinqiong) [reply matchzoo/utils/get_file.py]: I have fixed that.

File: matchzoo/utils/get_file.py (hunk 4 above)
Comment (caiyinqiong) [reply matchzoo/utils/get_file.py]: I have fixed it.

File: matchzoo/utils/get_file.py (hunk 5 above)
Comment (caiyinqiong) [reply matchzoo/utils/get_file.py]: The ',' will print a space, so it doesn't need a space anymore.

File: tests/test_utils.py [hunk 6]
```diff
@@ -54,3 +56,13 @@ def test_get_file():
...
+    file_hash = utils._hash_file(file_path, algorithm='md5')
+
+    file_path2 = utils.get_file(
+        'LICENSE', _url, extract=False,
+        file_hash=file_hash,
+        cache_dir=matchzoo.USER_DATA_DIR,
+        cache_subdir='LICENSE'
+    )
+    file_hash2 = utils._hash_file(file_path2, algorithm='md5')
+    assert file_hash == file_hash2
```
Comment (Chriskuei) [inline tests/test_utils.py]: Delete test file in the end

**Pull request merged**